app/controllers/admin/pods_controller.rb

```diff
@@ -0,0 +1,21 @@
+
+module Admin
+  class PodsController < AdminController
+    respond_to :html, :json
+
+    def index
+      pods_json = PodPresenter.as_collection(Pod.all)
+
+      respond_with do |format|
+        format.html do
+          gon.preloads[:pods] = pods_json
+          gon.preloads[:unchecked_count] = Pod.unchecked.count
+          gon.preloads[:error_count] = Pod.check_failed.count
+
+          render "admins/pods"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.